pkg/pb/notifications.proto

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package notifications;
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+message Notification{
+    enum Scope {
```

Comment:
Use a string for scopes so it's future compatible.

Reply:
how does a string make it more future compatible? if an unknown string comes in, and the SNO has a language other than english selected, what do we show? how is an unknown string different than an unknown enum value? aren't protobuf enums future compatible in the same sense as strings would be?

Reply:
We should assume that storage nodes will end up lagging behind compared to the notifications we will send them. So when we introduce a new scope then the handling code for the scope will fail, rather than defaulting to English.